homeassistant/components/knx/services.yaml

```diff
@@ -24,3 +24,25 @@ event_register:
       example: "1/1/0"
     remove:
       description: "Optional. If `True` the group address will be removed. Defaults to `False`."
+exposure_register:
+  description: "Add or remove exposures to KNX bus. Only exposures added with this service can be removed."
```

Comment:
The yaml schema has recently received an update.  Please check core platforms eg `fan`, `light`, etc for the new options.

Reply:
Do you mean this? https://github.com/home-assistant/core/pull/46410

Reply:
Its for the `services.yaml` file

Reply:
Ah now I see. 
I have tried to stick to these examples, but on my test instance no different view is rendered, except that I deleted "Optional" in the description thinking the required key would be shown. Is this still work in progress?